redirects.txt

```diff
@@ -205,8 +205,9 @@ applications/general/multi_companies/manage_multi_companies.rst applications/gen
 applications/general/payment_acquirers/payment_acquirers.rst applications/general/payment_acquirers.rst   # payment_acquirers/payment_acquirers -> payment_acquirers
 applications/general/unsplash/unsplash_access_key.rst applications/general/unsplash.rst                   # unsplash/unsplash_access_key -> unsplash
 applications/general/unsplash/unsplash_application_id.rst applications/general/unsplash.rst               # unsplash/unsplash_application_id -> unsplash
-applications/general/base_import/import_faq.rst applications/general/export_import_data.rst               # base_import/import_faq -> odoo_basics/import_data
-applications/general/base_import/adapt_template.rst applications/general/export_import_data.rst           # base_import/adapt_template -> odoo_basics/import_data
+applications/general/base_import.rst applications/general/export_import_data.rst                          # base_import -> export_import_data
+applications/general/base_import/import_faq.rst applications/general/export_import_data.rst               # base_import/import_faq -> export_import_data
+applications/general/base_import/adapt_template.rst applications/general/export_import_data.rst           # base_import/adapt_template -> export_import_data
```

Comment:
I don't get why we have three lines of diff if there is only one rule added (but I don't see the diff for the two previous lines :confused: ).

Reply:
it's what comes after the `#` that was wrong. Just a comment update :)